unitTests/ui/MLFacet/MLFacetTest.ts

```diff
@@ -29,6 +31,10 @@ export function MLFacetTest() {
       expect(qsmValues).toEqual(test.cmp.values.selectedValues);
     }
 
+    function simulateRequest() {
```

Comment:
Rather than putting `facetRequest` in an outer scope, could it be better to change this function to return it and rename to `getFirstFacetRequest`?